### ISSUE TITLE
Is there a newer version?

### ISSUE THREAD

**Issue by robbrandt** [eileenmcnaughton/nz.co.fuzion.membershiprenewalcontrol#1]
Eileen, is this the most recent version you have of this?  Something  seems amiss.  We are trying to use it in a 4.6 site, and membership creation fails because the contact_id isn't present (civi thinks it's just updating an existing membership record).   I modified it to pull the contact_id along with the other existing membership info, but I am wondering if there are other issues as well that I haven't discovered yet.

**Comment by eileenmcnaughton** [comment]
yes - I think it is - it's really quite customer specific - ie. there is hard coding. But it's good to use as a starting point
